data/seasons/all-watermarks.json

```diff
@@ -1,54 +1,55 @@
 [
-  "/common/destiny2_content/icons/02478e165d7d8d2a9f39c2796e7aac12.png",
```

Comment:
Not sure I fully understand how these are generated, but should they be sorted or something for better diffs?

Reply:
These are sorted already, the diff really just reflects that Bungie changed a ton of URLs.

Reply:
Wow. Well that's why we don't hard code them.